features/post/post-card/post-info-section/post-info-section.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { ReactNode } from 'react';
+
+import { BookmarkOutline } from '@/assets/icons/bookmark-outline';
+import { HeartFilled } from '@/assets/icons/heart-filled';
+import { HeartOutlined } from '@/assets/icons/heart-outlined';
+import { PaperPlaneOutline } from '@/assets/icons/paper-plane-outline';
+import { DateFormatter, useDateFormat } from '@/shared/hooks/useDateFormat';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, IconButton, Typography } from '@/shared/ui';
+import { useRouter } from 'next/router';
+
+import s from './post-info.module.scss';
+
+type PostInfoProps = {
+  actions?: ReactNode;
+  createdAt: string;
+  icons?: {
+    avatar: string;
+    userName: string;
+  }[];
+  likesCount: number;
+};
+
+export const PostInfoSection = (props: PostInfoProps) => {
+  const { actions, createdAt, icons, likesCount } = props;
+  const { locale } = useRouter();
+  const { t } = useTranslation();
+
+  const customDateTimeFormatter: DateFormatter = parts => {
+    const date = new Date(createdAt);
+
+    return new Intl.DateTimeFormat(locale, {
+      day: 'numeric',
+      month: 'long',
+      year: 'numeric',
+    }).format(date);
+  };
+
+  const formattedDate = useDateFormat(
+    { day: 'numeric', month: 'long', year: 'numeric' },
+    customDateTimeFormatter
+  );
+
+  const formatNumberWithSpaces = (num: number | string): string => {
+    return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ' ');
+  };
+
+  const getInitials = (userName: string): string => {
```

Comment:
This function should be placed in a separate file for reuse. It is better to give a more abstract name, for example `getAcronym`. It may be useful to be able to set the number of capital letters in the final string (not everywhere the first 2 letters are necessary).